src/scripts/services/dimStoreService.factory.js

```diff
@@ -21,13 +21,28 @@ function StoreService(
   dimManifestService,
   $translate,
   uuid2,
-  dimFeatureFlags
+  dimFeatureFlags,
+  dimSettingsService,
+  $http
 ) {
   var _stores = [];
   var _idTracker = {};
 
   var _removedNewItems = new Set();
 
+  var dimClassifiedData;
+  var dimClassifiedDataPromise;
+
+  if ($DIM_FLAVOR === 'dev') {
+    dimClassifiedData = require('app/data/classified.json');
```

Comment:
This should probably just be a different URL for the $http.get, rather than a `require` - that way it'll behave the same.